ramls/types/configuration/jsonapiError.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I think its beneficial to move this schema to ramls/types directory since I foresee this being referenced and used in several error responses for different endpoints, not just configuration.